src/fiat-connect-client.ts

```diff
@@ -239,25 +236,25 @@ export class FiatConnectClient implements FiatConnectApiClient {
   }
 
   /**
-   * https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#3311-get-quotein
+   * https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#3411-post-quotein
    */
-  async getQuoteIn(
-    params: QuoteRequestQuery,
+  async addQuoteIn(
```

Comment:
the name is a little clunky; I'd consider "createQuote", but I don't think "getQuote" is good anymore since it's calling a POST endpoint